Source/WebGPU/WGSL/TypeDeclarations.rb

```diff
@@ -8,13 +8,22 @@
 }
 
 operator :*, {
+    [T < Number, N].(Vector[T, N], T) => Vector[T, N],
+    [T < Number, N].(T, Vector[T, N]) => Vector[T, N],
+    [T < Number, N].(Vector[T, N], Vector[T, N]) => Vector[T, N],
```

Comment:
aside: I checked the spec, but component-wise multiplication of vectors seems like a strange operation.